terraform-aws/LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright (c) 2015, Travis Cline <travis.cline@gmail.com>
```

Comment:
I think we want everything under kubernetes to be covered by the Kubernetes apache license and copyright the kubernetes owners.

If that's not ok, this probably belongs in a different repo.